site/docs/v1/tech/core-concepts/entity-management.adoc

```diff
@@ -0,0 +1,133 @@
+---
+layout: doc
+title: Entity Management
+description: An overview of FusionAuth Entity Management
+---
+
+:sectnumlevels: 0
+
+== Overview
+
+There are many use cases where it is helpful to model entities in addition to users. Examples might include devices, cars, computers, customers, companies, etc. +
+
+[%hardbreaks]
+
+_Enter Entities._ Entities allow you to model everything right from FusionAuth!
+
+* <<Features>>
+* <<Common Applications>>
+* <<Entity Types Homepage>>
+* <<Entity Homepage>>
+
+=== Features
+
+==== Scalability
+- FusionAuth Entity Management supports large volumes of Entities right out of the box.
+- This is especially helpful for Internet of Things (IOT) devices; FusionAuth scales right alongside them.
+
+==== Typecasting
+- Entities can have types.
+- For example, `locks`, `cars`, `companies`, `corporate divisions`, `computers`, `APIs`
+- Entity Types can help define permissions.
+- You are limited only by your business need and imagination!
+
+==== Permissions aware
+- Permissions can be assigned to each Entity Type.
+- Entities can share permissions with other Entities (In OAuth parlance, this synonymous with the client credentials grant).
+- Users can have permissions to Entities.
+
+=== Common Applications
+- Corporate relationship modeling
+- Per use device permissions
+- Internet IoT
```

Comment:
I think a use case here would be helpful.

Maybe just take the diagram and use case from "Permission retrieval examples for entities (via Client Credentials Grant)" here: https://docs.google.com/drawings/d/1si4nidqeBfPhk6XRp9vuQZq5rvtj6FMc00crldrpY8k/edit ?